app/model/schema/token_bond.py

```diff
@@ -126,8 +128,11 @@ class RetrieveStraightBondTokenResponse(BaseModel):
     interest_payment_date10: Optional[str]
     interest_payment_date11: Optional[str]
     interest_payment_date12: Optional[str]
+    interest_payment_currency: str
     redemption_date: str
     redemption_value: int
+    redemption_value_currency: str
+    base_fx_rate: float
```

Comment:
Added currency attributes to response schema. (`face_value_currency`・`interest_payment_currency`・`redemption_value_currency`・`base_fx_rate`)
And no attributes added to request query schema because it seems that query params of currency are not needed in the foreseeable future